cmpirque/users/admin.py

```diff
@@ -0,0 +1,4 @@
+from django.contrib import admin
+from .models import User
+
+admin.site.register(User)
```

Comment:
you missing the blank line at the end

Reply:
i will put the blank line at the end

yes, i have checked the local settings according to documentation and it's ok

Reply:
Ok! and the tests passes?